platform-api/tests/test_discord.py

```
@@ -0,0 +1,147 @@
+from dependencies import get_settings
+from models.settings import WorkerSettings
+
+class MockKV:
+    def __init__(self):
+        # Pre-seed our mock KV with one known channel
+        self.store = {"discord-webhook:announcements": "http://fake-discord.url"}
+        
+    async def get(self, key):
+        return self.store.get(key)
+        
+    async def put(self, key, value, expirationTtl=None):
```

Ruff (0.15.11)

[warning] Unused method argument: `expirationTtl`

(ARG002)